src/buttonGroup/ButtonGroup.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { cloneElement } from 'react'
```

Comment:
Could we rename this file index.tsx? To avoid doubling name in the import?
`import { ButtonGroup } form `@blablacar/ui-library/...../buttonGroup/ButtonGroup`
vs
`import { ButtonGroup } form `@blablacar/ui-library/...../buttonGroup`

Wdyt?

Reply:
Usually, `index.js` is used only to export files. I'm ok on keeping it named as the component name.

@simonrelet  what are the best practices on exporting files?

Reply:
I think this was to avoid breaking changes, but I agree on the long term, we should need only one file. However if we do this refactoring on all the files it could be nice to batch the breaking changes into one PR. I don't know what you have in mind @gabrielagabriel for that?

Reply:
yes.. this avoids breaking changing because for now we are only updating the documentation. I rather move files on another PR

Reply:
> @gabrielagabriel 
> @simonrelet what are the best practices on exporting components?

What I think is:

- Imports ending with `/buttonGroup/ButtonGroup` are probably mistakes, I don't think components were ment to be imported without their styles, that being said, it wasn't clearly explain what is public and what is internal, so it can be considered a braking change.

- I think we should only be allowed to import from the package root: `import { ButtonGroup } from '@blablacar/ui-library'`, the internal structure is not part of the public API and tree shaking is [not a problem with these imports/exports with Webpack](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free). As it would require a longer change (for a next time), I think we should now have a minimal path: `/buttonGroup`. The index file either containing the component or just import/export would work, I have a small preference for the second one (just import/export) as it ease discoverability and is more consistent:
  ```
  ┬╴ buttonGroup/
  ├╴ index.tsx
  ├╴ ButtonGroup.tsx       ⬅ Component code here.
  ├╴ ButtonGroup.test.tsx
  ├╴ ButtonGroup.story.mdx
  └╴ ...
  ```